docs/source/glossary.md

```diff
@@ -47,4 +47,13 @@ Posterior
   Once you are satisfied with the model, posterior distribution can be summarized and interpreted. Common questions for the posterior include: intervals of defined boundaries, intervals of defined probability mass, and point estimates. When the posterior is very similar to the prior, the available data does not contain much information about a parameter of interest.
 
   - For more on generating and interpreting the posterior samples, see Statistical Rethinking 2nd Edition By Richard McElreath, chapter 3.
+
+Maximum a Posterior 
```

Comment:
Posterior should be `Posteriori`

Reply:
@martinacantaro Hi. Martina. Thanks. I've fixed the typo